### PR TITLE
{vis}[foss/2017b] scikit-image v0.13.1, matplotlib v2.1.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2017b-Python-3.6.3.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    ('matplotlib', '2.1.1', versionsuffix),
+    ('Pillow', '4.3.0', versionsuffix),
+]
+
+exts_list = [
+    ('networkx', '1.11', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/networkx'],
+        'checksums': ['0d0e70e10dfb47601cbb3425a00e03e2a2e97477be6f80638fef91d54dd1e4b8'],
+    }),
+    (name, '0.13.0', {
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2017b-Python-3.6.3.eb]: shouldn't it be `'0.13.1'`, or better still, `version`?

Comment (boegel) [reply easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2017b-Python-3.6.3.eb]: Oh snap, good catch!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2302.phanpy.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d503dd7a74f94fbf30481e0d6a5878b8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2035.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/230339e3f98f8cd4b6849ab9ca1ff4f3 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.6.6
See https://gist.github.com/9c9a4707b77f38f5e49c7907a45d4343 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**